client/__tests__/e2e/puppeteerUtils.js

```diff
@@ -231,53 +256,82 @@ export const cellxgeneActions = puppeteerPage => ({
 
   async createCategory(categoryName) {
     await puppeteerUtils(puppeteerPage).clickOn("open-annotation-dialog");
-    await puppeteerUtils(puppeteerPage).typeInto("new-category-name", categoryName);
+    await puppeteerUtils(puppeteerPage).typeInto(
+      "new-category-name",
+      categoryName
+    );
     await puppeteerUtils(puppeteerPage).clickOn("submit-category");
   },
 
   async renameCategory(oldCatgoryName, newCategoryName) {
-    await puppeteerUtils(puppeteerPage).clickOn(`${oldCatgoryName}:see-actions`);
-    await puppeteerUtils(puppeteerPage).clickOn(`${oldCatgoryName}:edit-category-mode`);
-    await puppeteerUtils(puppeteerPage).clearInputAndTypeInto(`${oldCatgoryName}:edit-category-name-text`, newCategoryName);
-    await puppeteerUtils(puppeteerPage).clickOn(`${oldCatgoryName}:submit-category-edit`);
+    await puppeteerUtils(puppeteerPage).clickOn(
+      `${oldCatgoryName}:see-actions`
+    );
+    await puppeteerUtils(puppeteerPage).clickOn(
+      `${oldCatgoryName}:edit-category-mode`
+    );
+    await puppeteerUtils(puppeteerPage).clearInputAndTypeInto(
+      `${oldCatgoryName}:edit-category-name-text`,
+      newCategoryName
+    );
+    await puppeteerUtils(puppeteerPage).clickOn(
+      `${oldCatgoryName}:submit-category-edit`
+    );
   },
 
   async deleteCategory(categoryName) {
     await puppeteerUtils(puppeteerPage).clickOn(`${categoryName}:see-actions`);
-    await puppeteerUtils(puppeteerPage).clickOn(`${categoryName}:delete-category`);
+    await puppeteerUtils(puppeteerPage).clickOn(
+      `${categoryName}:delete-category`
+    );
   },
 
   async createLabel(categoryName, labelName) {
     await puppeteerUtils(puppeteerPage).clickOn(`${categoryName}:see-actions`);
-    await puppeteerUtils(puppeteerPage).clickOn(`${categoryName}:add-new-label-to-category`);
-    await puppeteerUtils(puppeteerPage).typeInto(`${categoryName}:new-label-name`, labelName);
+    await puppeteerUtils(puppeteerPage).clickOn(
```

Comment:
What line width are you using in Prettier? In the python code, we've been using 120 characters.

Reply:
AFAIK we use the default prettier settings which I see as 80.  Maybe in the future we can add a prettier config file

Reply:
AFAIK, we use the default prettier config (i,e, we implicitly have a config - the default).   I believe 80 is the default line length.